experiments/conductor/cmd/runner/utilities.go

```diff
@@ -296,61 +298,90 @@ func executeCommand(opts *RunnerOptions, cfg CommandConfig) (string, string, err
 		}
 	}
 
-	log.Printf("Starting command step: %s", cfg.Name)
-	log.Printf("[%s] working directory: %s", cfg.Name, cfg.WorkDir)
-	log.Printf("[%s] command: %s %s", cfg.Name, cfg.Cmd, strings.Join(cfg.Args, " "))
-	if cfg.Stdin != nil {
-		log.Printf("[%s] stdin: %s", cfg.Name, cfg.Stdin)
+	if cfg.RetryBackoff == 0 {
+		cfg.RetryBackoff = time.Second
```

Comment:
Nit: If this is related to quota then I guess one second of backoff time may not be sufficient. But we can improve it later.

Reply:
changed it to 30s for generative commands.